index.html

```diff
@@ -13,24 +13,25 @@
     <section>
 
         <form>
+            <h1>Stutern Feedback Form</h1>
             <div class="form-control">
-                <label>First name </label>
+                <label><b>First name</b></label>
```

Comment:
yes, using a <b> tag works.
However, with CSS, you can make this bold using this property
`font-weight` which takes values like bold, bolder etc

See this link for more information

https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#:~:text=Meaning%20of%20relative%20weights,-When%20lighter%20or&text=Note%20that%20when%20using%20relative,purposes%20of%20relative%20weight%20calculation.